docs/src/prototypes/CopyToClipboard/index.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from 'react'
+import { Flex, Provider, Text, Button, Menu } from '@stardust-ui/react'
+import CopyToClipboard from './CopyToClipboard'
+import { PrototypeSection, ComponentPrototype } from '../Prototypes'
+import themeOverrides from './themeOverrides'
+import { NotificationProvider } from './NotificationProvider'
+
+type CopyToClipboardPrototypeProps = {
+  value: string
+  attached?: boolean
+}
+
+const CopyToClipboardPrototype: React.FC<CopyToClipboardPrototypeProps> = props => {
+  return (
+    <Flex gap="gap.medium" vAlign="center" padding="padding.medium">
+      <Text content="Commit: " />
+      <Text content={props.value} color="brand" />
+
+      <CopyToClipboard
+        attached={props.attached}
+        pointing
+        value={props.value}
+        trigger={<Button iconOnly icon="clipboard-copied-to" />}
+      />
+    </Flex>
+  )
+}
+
+const CopyToClipboardInMenu: React.FC = props => {
+  const items = [
+    {
+      key: 'edit',
+      content: 'Edit',
+      menu: [
+        'Open File...',
+        'Save File...',
+        render =>
+          render('Copy text', (Component, props) => {
+            return (
+              <CopyToClipboard
+                position="after"
+                align="bottom"
+                pointing
+                value={'Julius Caesar'}
+                trigger={<Component {...props} />}
+              />
+            )
+          }),
+      ],
+    },
+  ]
+  return <Menu items={items} />
+}
+
+const CopyToClipboardPrototypes: React.FC = () => {
+  const commitID = '3422f7d'
+  return (
+    <PrototypeSection title="Copy to Clipboard">
```

Comment:
can you please add a note to the beginning of the prototype page, saying:

Note: For screen reader users, make sure to use react-aria-live or similar library to announce the notification.